src/OrchardCore.Modules/OrchardCore.Contents/Indexing/DefaultContentIndexHandler.cs

```diff
@@ -7,47 +7,46 @@ public class DefaultContentIndexHandler : IContentItemIndexHandler
     {
         public Task BuildIndexAsync(BuildIndexContext context)
         {
-            // Text values are stored in field.keyword naturally
-            // to replicate Elasticsearch behavior
             context.DocumentIndex.Set(
                 IndexingConstants.ContentTypeKey,
                 context.ContentItem.ContentType,
-                DocumentIndexOptions.Analyze);
+                DocumentIndexOptions.Keyword | DocumentIndexOptions.Store);
```

Comment:
I have an idea for that matter. Basically, I reverted the content index setting. By default, everything was indexed as a keyword in Lucene. Now, everything will be indexed as a "full-text" and a "keyword" just like Elasticsearch does. So, basically, the current Queries that are built with Lucene might be broken after reindexing the indices. So, it would be possible to create a migration that would revert the content index settings as they were.

Reply:
Basically. If someone had checked the "Analyzed" option on a field; now it will be the default. 

So, instead, I added the opposite option which is "Keyword" so that the field be indexed as a single token explicitly. This means that "BlogPost.Subtitle" for example will become indexed as a "Keyword" and will not index a "BlogPost.Subtitle.keyword" field.

Right now, by default "BlogPost.Subtitle" is a "Keyword" in OC 1.3. and when we click on the option "Analyze" it makes the field a TextField (tokenized) ready for Full-Text search. Though, it was a bad assumption on our part to think that people will not want to tokenize their text field by default. This is what Lucene is meant for! So, just like Elasticsearch, when we set a custom mapping on a field, it sets this field to a single indexed type instead of indexing a TextField (tokenized) and a "Keyword".

Reply:
Done as part of [1a93824](https://github.com/OrchardCMS/OrchardCore/pull/11052/commits/1a938248d6c0f1a96fcf72d3bb8938712468421c)
Now the old Lucene settings are migrated to the new equivalent ones.
Elasticsearch now will set its content indexing settings to the Lucene ones on the first install.